update_cmake_build.bat

```diff
@@ -7,7 +7,7 @@ REM          Allows MMEX to be run successfilly within MSVC 2015 IDE
 REM --------------------------------------------------------------------------
```

Comment:
- Should be this file `update_cmake_build.bat` moved in the util folder `util/update_cmake_build.bat` where are other useful scripts located?
- One naming question. From the content of `update_cmake_build.bat` script the intent is to prepare project files for Microsoft Visual Studio. Therefore the script should be renamed to something like `update_cmake_build_for_msv.bat`?
- The same question applies also to script `update-po-files.sh` to move it to `util/update-po-files.sh`